students/DiannaTingg/Session02/grid_printer.py

```diff
@@ -0,0 +1,91 @@
+# Lesson 02 Exercise: Grid Printer
+
+# Part 1: Write a function that prints a grid like the example.
+
+
+def standard_grid():
+    # Top row
+    print("+ " + "- " * 4 + "+ " + "- " * 4 + "+")
+
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+
+    # Middle row
+    print("+ " + "- " * 4 + "+ " + "- " * 4 + "+")
+
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+    print("| " + " " * 8 + "| " + " " * 8 + "|")
+
+    # Bottom row
+    print("+ " + "- " * 4 + "+ " + "- " * 4 + "+")
+
+
+# Test
+print("Standard Grid")
+standard_grid()
+print()
+
+# Part 2: Write a function that prints the grid size based on an integer.
+
+
+def print_grid(n):
+    # Figure out what half of the grid should be
+    half = n // 2
+
+    # Top row
+    print("+ " + "- " * half + "+ " + "- " * half + "+")
+
+    for i in range(half):
+        print("| " + " " * (half * 2) + "| " + " " * (half * 2) + "|")
+
+    # Middle row
+    print("+ " + "- " * half + "+ " + "- " * half + "+")
+
+    for i in range(half):
+        print("| " + " " * (half * 2) + "| " + " " * (half * 2) + "|")
+
+    # Bottom row
+    print("+ " + "- " * half + "+ " + "- " * half + "+")
```

Comment:
Top, middle and bottom row are the same, yes? 

Why not define them in one place and re-use?